invenio_rest/ext.py

```diff
@@ -105,3 +106,21 @@ def init_config(self, app):
         app.config.setdefault('REST_ENABLE_CORS', False)
         app.config.setdefault('RATELIMIT_GLOBAL', '5000/hour')
         app.config.setdefault('RATELIMIT_HEADERS_ENABLED', True)
+
+    def limit_per_user(self, limit_value, per_method=False, methods=None,
```

Comment:
@SamiHiltunen we might need to use similar pattern as we use in Flask-Menu. Because usually you do not have access to an instance of InvenioLimitter.